### PR TITLE
Front end for admin page #77

### LINKED ISSUE #10: As an Administrator I want to be able to assign patients to doctors so that every patient has a doctor.

### PR DESCRIPTION
**Pull request by Cresfault**
Back end for admin page role approval added. Completes task #38 and #22

### CODE REVIEW

**Review by Omarelk09 — DISMISSED** [review]

File: server.js [hunk 1]
```diff
@@ -6,6 +6,12 @@ if (process.env.NODE_ENV !== 'production') {
 //importing
 const express = require("express")
 const app = express()
+
+app.use(express.urlencoded({ extended: false }))
+
+//initializing db 
+const db = require('c:/Users/moham/OneDrive/Desktop/Winter2022/twat1/soen390/database')
```
Comment (gfdb) [inline server.js]: this is wrong

Comment (gfdb) [reply server.js]: line 13 should be const db = require('./database')

**Review by maritabrichan — DISMISSED** [review]

**Review by gfdb — CHANGES_REQUESTED** [review]

File: server.js (hunk 1 above)
Comment (Cresfault) [reply server.js]: The path is changed

**Review by gfdb — APPROVED** [review]

**Review by david-roper — APPROVED** [review]

**Review by maritabrichan — APPROVED** [review]

**Review by Omarelk09 — APPROVED** [review]
good job

**Pull request merged**